lib/univalue_get.cpp

```diff
@@ -59,6 +59,22 @@ bool ParseInt64(const std::string& str, int64_t *out)
         n <= std::numeric_limits<int64_t>::max();
 }
 
+
+bool ParseUint64(const std::string& str, uint64_t *out)
```

Comment:
Could be `static`.

Reply:
there is no difference either way. not worth changing